04/app.js

```diff
@@ -1,8 +1,40 @@
-console.log('DOM');
+console.log("DOM");
 
 // struktura do wykorzystania w pętli
 const menuItems = [
-    {text: 'start', url: '/'},
-    {text: 'galeria', url: '/gallery'},
-    {text: 'kontakt', url: '/contact'},
-];
\ No newline at end of file
+  { text: "start", url: "/" },
+  { text: "galeria", url: "/gallery" },
+  { text: "kontakt", url: "/contact" },
+];
+
+const nav = document.querySelector("nav");
+
+if (nav) {
+  const ul = document.createElement("ul");
+  nav.appendChild(ul);
+  if (ul) {
```

Comment:
Jeśli linijkę wyżej tworzysz `ul` to nie ma sensu sprawdzać czy istnieje - jeśli nie zostałby utworzony to będzie rzucony błąd :)